src/handbook/peopleops/coaching-plans.md

```diff
@@ -0,0 +1,106 @@
+role changes, and other changes.
+
+## When to Use a Coaching Plan
+
+A coaching plan should be initiated by a team members manager when:
+
+- A team member shows potential for growth
+- The growth area is specific and coachable
+- The identified area for growth isn't a role deficiency, and a [PIP][pip] is in
+  order
+  - Minor performance areas need attention but don't warrant formal PIP
+    intervention
+- A team member expresses interest in expanding their responsibilities
+- The team member is to be prepared for future role progression
+- You observe behaviors or skills that could benefit from focused development
+
+## Coaching Plan vs. Performance Improvement Plan
+
+**Coaching Plans** are:
+
+- **Proactive** - Focused on development and growth
+- **Supportive** - Emphasize mentorship and guidance
+- **Opportunity-based** - Address areas for improvement before they become
+  problems
+- **Growth-oriented** - Aimed at expanding capabilities and preparing for
+  advancement
+- **Collaborative** - Employee actively participates in goal-setting
+
+**Performance Improvement Plans** are:
+
+- **Reactive** - Address existing performance deficiencies
+- **Formal** - Structured process with documented consequences
+- **Problem-focused** - Correct performance that falls below standards
+- **Corrective** - Aimed at bringing performance up to acceptable levels
+- **Prescriptive** - Clear expectations with defined outcomes
+
+## Creating a Coaching Plan
+
+### Identify the Opportunity
+
+Managers should:
+
+- Observe areas where the employee could grow or develop
+- Consider the employee's career aspirations and interests
+- Assess skills that would benefit from strengthening
+- Identify opportunities that align with business needs
+- Discuss and align with their own manager on the growth opportunity
+
+### Develop the Coaching Plan
+
+The coaching plan should include:
+
+1. Development area: What skill or such is to be improved.
+1. Current state: Managers' assesment of the current level.
+1. Goal state: A [SMART goal][smart] goal to collectly aim for.
+1. Activities: What actions, training, experience are to be done to achieve the
+   set goal.
+1. Timeline: Start and end date, typically 3 months apart.
+1. Support assets: What tools, meeting, or manager assitances will be provided
+1. Update Cadance: How manager and team member align on progress, and how to
+   collectively achieve goals.
+
+## Documentation and Tracking
+
+While coaching plans are less formal than PIPs, it's important to:
+
+- Document the coaching plan and goals
+- Track progress and development activities
+- Note feedback and adjustments made
+- Record achievements and milestones
+- Maintain confidentiality while sharing appropriate updates with HR and
+  management
+  - Coaching plans are only shared with; the team member, manager, the managers'
+    manager
+
+## Transitioning Between Coaching Plans and PIPs
```

Comment:
The rest of this read excellently, but this section just makes it feel like a PIP v0.5 again.

Whilst a PIP may be warranted after coaching plans fail (when deployed due to deficiency in performance of an expected duty) I'm not sure it needs calling out explicitly. 

There are many reasons for coaching plans to be deployed (e.g. me taking on EM responsibility, or someone trying out a new role temporarily and us making sure support is in place) that have no relevance to this section where a PIP would not be an appropriate consequence of falling short in a coaching plan